pkgs/applications/science/math/readstat/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, gettext, libtool }:
+
+stdenv.mkDerivation rec {
+  name = "readstat";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner = "WizardMac";
+    repo = "ReadStat";
+    rev = "v${version}";
+    sha256 = "1r04lq45h1yn34v1mgfiqjfzyaqv4axqlby0nkandamcsqyhc7y4";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook libtool gettext ];
```

```suggestion
  nativeBuildInputs = [ pkg-config autoreconfHook ];
```
Those are already included in autoreconfHook

---

Ah, thanks.